sdk/src/types/block/core/block.rs

```diff
@@ -154,17 +173,18 @@ impl Block {
     /// The maximum number of bytes in a block.
     pub const LENGTH_MAX: usize = 32768;
 
-    /// Creates a new [`Block`].
-    #[inline(always)]
-    pub fn new(header: BlockHeader, body: BlockBody, signature: impl Into<Signature>) -> Self {
-        let signature = signature.into();
+    // /// Creates a new [`Block`].
+    // #[inline(always)]
+    // pub fn new(
+    //     header: BlockHeader,
+    //     body: BlockBody,
+    //     signature: impl Into<Signature>,
+    //     params: &ProtocolParameters,
+    // ) -> Result<Self, Error> {
 
-        Self {
-            header,
-            body,
-            signature,
-        }
-    }
+    // }
+
+    // TODO add new_unverified
```

Comment:
Want to do in this PR or another one?

Reply:
No actually I forgot to remove the comment, we decided with Chloé to do with `UnsignedBlock::finish` completely and remove `Block::new`

Reply:
Done